src/Stratis.Bitcoin.Features.Consensus/ProvenBlockHeaders/IProvenBlockHeaderStore.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus.ProvenBlockHeaders
+{
+    /// <summary>
+    /// Cache layer for <see cref="ProvenBlockHeader"/>s.
+    /// </summary>
+    public interface IProvenBlockHeaderStore : IProvenBlockHeaderProvider, IDisposable
+    {
+        /// <summary>
+        /// Adds a <see cref="ProvenBlockHeader"/> to pending batch concurrent dictionary.
+        /// </summary>
+        /// <param name="provenBlockHeader">A <see cref="ProvenBlockHeader"/> item to add.</param>
+        /// <param name="newTip">Block hash and height pair associated against the <see cref="ProvenBlockHeader"/> item.</param>
```

Comment:
hash and height pair that represent the tip of <see= IProvenBlockHeaderStore

Reply:
not fixed

Reply:
Changed to  `/// <param name="newTip">Hash and height pair that represent the tip of <see cref="IProvenBlockHeaderStore"/>.</param>`